test/jest/components/PagesList.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import PagesList, { Page } from 'components/PagesList';
+import Consts from 'utils/consts';
+
+describe('(Component) PagesList', () => {
+    // @ts-ignore To make typescript happy
+    const noop = (..._: any[]) => {};
+    const defaultProps = {
+        onPageSelected: noop,
+        onPageRemoved: noop,
+        onPageReorder: noop,
+        pages: [],
+        selected: undefined,
+        isEditMode: false
+    };
+    const samplePages: Page[] = [
+        { id: 'abafar', name: 'Abafar', isDrillDown: false, tabs: [], widgets: [] },
+        { id: 'mustafar', name: 'Mustafar', isDrillDown: false, tabs: [], widgets: [] }
+    ];
+
+    it('should renders component', () => {
```

Comment:
Minor thing: probably more correct would be:
```suggestion
    it('should render component', () => {
```